lib/Doctrine/ORM/Tools/Export/Driver/XmlExporter.php

```diff
@@ -451,4 +454,61 @@ private function _asXml($simpleXml)
 
         return $dom->saveXML();
     }
+
+    /**
+     * @param ClassMetadataInfo $metadata
+     * @param SimpleXMLElement  $root
+     */
+    private function processEntityListeners(ClassMetadataInfo $metadata, SimpleXMLElement $root)
+    {
+        if (0 !== \count($metadata->entityListeners)) {
+            $entityListenersXml = $root->addChild('entity-listeners');
+            $entityListenersXmlMap = [];
+
+            $this->generateEntityListenerXml($metadata, $entityListenersXmlMap, $entityListenersXml);
+        }
+    }
+
+
+    /**
+     * @param ClassMetadataInfo $metadata
+     * @param array             $entityListenersXmlMap
+     * @param SimpleXMLElement  $entityListenersXml
+     */
+    private function generateEntityListenerXml(ClassMetadataInfo $metadata, array $entityListenersXmlMap, SimpleXMLElement $entityListenersXml)
```

Comment:
` : void`. You can also remove docblocks that are redundant to the type declarations.